.github/workflows/main.yml

```diff
@@ -166,3 +166,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          args: >
+            -Dproject.settings=.sonar-project.properties
```

Comment:
Do we really have to split the args line?  The YAML folded style is nice for multiple arguments since the indention in each line will be ignored and a line break will be inserted at the end, but I'd rather see one line when we have one argument:

```yml
          args: -Dproject.settings=.sonar-project.properties
```

Reply:
Works for me.